pykusto/_src/query.py

```diff
@@ -619,12 +624,28 @@ def by(self, *args: Union[_AssignmentToSingleColumn, BaseColumn, BaseExpression]
             self._by_assignments.append(_AssignmentToSingleColumn(_AnyTypeColumn(column_name), group_exp))
         return self
 
+    def shuffle_key(self, *keys: Union[_AssignmentToSingleColumn, BaseColumn, BaseExpression]):
+        if keys is not None:
+            for key in keys:
+                self._shuffle_keys.append(key)
+
+        self._shuffle_by_keys = True
+        return self
+
     def _compile(self) -> KQL:
-        result = f"summarize {', '.join(a.to_kql() for a in self._assignments)}"
```

Comment:
Don't delete the `"summarize"`. leave it here, and only add the hints in a function like `_compile_shuffle_hints()`

Reply:
A redundant space fails the query?

Reply:
I changed it, I think it looks better now.
I'll push once we resolve the other discussions.

Reply:
pushed